packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -1900,6 +1902,184 @@ describe("init", () => {
 			});
 		});
 	});
+
+	describe("from dashboard", () => {
+		mockApiToken();
+		mockAccountId({ accountId: "LCARS" });
+		afterEach(() => {
+			unsetAllMocks();
+		});
+		const mockDashboardScript = `
+		export default {
+			async fetch(
+				request: Request,
+				env: Env,
+				ctx: ExecutionContext
+			): Promise<Response> {
+				return new Response("Hello World!");
+			},
+		};
```

Comment:
I don't think it matters too much since we don't actually do anything with these scripts, but the dashboard script will always be JS, right?

Reply:
Ahahahaha fair point.